index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name = "viewport" content="width=device-width, initial-scale=1">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="headerbackground">
+        <div class="header">
+            <h1 class="Logo">My Logo</h1>
+
+            <nav class="navlinks">
+                 <ul class="navitems">
+                    <li class="vector1"><a href="#"><img src="Images/Vector1.png" alt="shape"></a></li>
+                    <li class="vector2"><a href="#"><img src="Images/Vector2.png" alt="shape"></a></li>
+                    <li class="vector3"><a href="#"><img src="Images/Vector3.png" alt="shape"></a></li>
+                </ul>
+            </nav>
+        </div>
+        <div class="headline">
+            <ul class="images">
+                <li class="shape1"><img src="Images/Ellipse8.png" alt="shape"></li>
+                <li class="shape2"><img src="Images/Ellipse11.jpg" alt="shape"></li>
+                <li class="shape3"><img src="Images/Rectangle7.png" alt="shape"></li>
+                <li class="shape4"><img src="Images/Ellipse5.png" alt="shape"></li>
+                <li class="shape5"><img src="Images/Rectangle4.png" alt="shape"></li>
+                <li class="shape6"><img src="Images/Ellipse3.png" alt="shape"></li>
+            </ul>
+
+            <h2 class="header2">I’m Iptisam 
+                Glad to see you!</h2>
```

Comment:
### Your Hero TItle
![heading-fixable](https://user-images.githubusercontent.com/93039359/185140749-ac92b29c-cee2-4994-8623-a86db19b3b9d.png)

### Figma Hero Title
![figma-heading](https://user-images.githubusercontent.com/93039359/185140771-e61ca04f-d828-4182-adfa-061964e3993b.png)


- [ ] Your hero title is not very similar to the Figma hero title :thinking: Please update your styling of your hero title to make it look similar to the Figma template :+1:
- [ ] Please use `h1` tag instead of `h2` to follow the [heading hierarchy](https://www.w3.org/WAI/test-evaluate/preliminary/#headings) :+1:

Reply:
I have a question in this part 
I have tried to edit multiple time the font style for this title but i dont get the same the exact as Figma
The font-family used is Poppins and in my CSS its also Poppins but no change. @HAMMAS-SALEEM